client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx

```diff
@@ -76,24 +73,12 @@ export default function ItemPreviewPaneHeader( {
 						</div>
 					</div>
 					<div className="item-preview__header-actions">
-						{ itemData.adminUrl ? (
-							<>
-								<Button
-									onClick={ closeItemPreviewPane }
-									className="item-preview__close-preview-button"
-									variant="secondary"
-								>
-									{ translate( 'Close' ) }
-								</Button>
-								<Button
-									variant="primary"
-									className="item-preview__admin-button"
-									href={ `${ adminUrl }` }
-									ref={ focusRef }
-								>
-									{ adminLabel }
-								</Button>
-							</>
+						{ extraProps?.headerButtons ? (
```

Comment:
hmm I see so extraProps are only used on the sites dashboard v2 as is. And it sounds like a4a has been tested already, and jetpack cloud should be the same, although I haven't yet worked out how to test `/sites` on jetpack cloud.